libraries/AP_HAL_ChibiOS/hwdef/CUAV-Nora/hwdef.dat

```diff
@@ -241,9 +241,9 @@ BARO MS56XX SPI:ms5611_imu
 BARO MS56XX SPI:ms5611_board
 
 # three IMUs
+IMU Invensensev3 SPI:icm42688 ROTATION_PITCH_180_YAW_270
```

Comment:
for existing boards this would force users to recalibrate their IMUs
It may be better to set EK3_PRIMARY in defaults.parm for this board, which should achieve the same effect, but without causing a recal

Reply:
If users's Nora without 42688, Do they also need to recalibrate their IMUs?